daprdocs/content/en/developing-applications/building-blocks/service-invocation/howto-invoke-services-grpc.md

```diff
@@ -0,0 +1,247 @@
+---
+type: docs
+title: "How-To: Invoke services using gRPC"
+linkTitle: "How-To: Invoke services using gRPC"
+description: "Call between services using service invocation"
+weight: 3000
+---
+
+This article describe how to use Dapr to connect services using gRPC.
+By using Dapr's gRPC proxying capability, you can use your existing proto based gRPC services and have the traffic go through the Dapr sidecar. Doing so yields the following [Dapr Service Invocation]({{< ref service-invocation-overview.md >}}) benefits to developers:
+
+1. Mutual authentication
+2. Tracing
+3. Metrics
+4. Access lists
+5. Network level resiliency
+6. API token based authentication
+
+## Step 1: Run a gRPC server
+
+The following example is taken from the [hello world grpc-go example](https://github.com/grpc/grpc-go/tree/master/examples/helloworld).
+
+```go
```

Comment:
Are we okay with this being only for Go develoeprs?

Reply:
I made this comment further down, that we are ending up with these HowTo that are very specific for one language depending on who wrote it. We need to show how these fit with other languages also.

Reply:
This is not for Go developers only. gRPC supports 9 languages, we're not going to show examples in every language supported by gRPC. developers who arrive at this article already have their gRPC services written in the language of their choice.

I added a comment to specify the example is in Go but applies to every language, even though in my opinion this is very redundant.